js/languages/en-US.json

```diff
@@ -741,5 +741,991 @@
     "10": "(GMT +10:00) Eastern Australia, Guam, Vladivostok",
     "11": "(GMT +11:00) Magadan, Solomon Islands, New Caledonia",
     "12": "(GMT +12:00) Auckland, Wellington, Fiji, Kamchatka"
+  },
+  "countries": {
```

Comment:
yup, but can an individual language be split across multiple files. E.g. can we have en-US.json & en-US-countries.json?